snippets/example.py

```diff
@@ -0,0 +1,56 @@
+from dotenv import load_dotenv
+from pydantic import BaseModel, Field
+from pydantic_settings import BaseSettings, PydanticBaseSettingsSource
+
+from pydantic_secrets import SecretManagerClientABC, SecretManagerSource
+
+
+class SecretManagerExample(SecretManagerClientABC):
```

Comment:
From my point of view realization of this manager  should be inside of library.

Reply:
This is just a code snippet - example of usage

Reply:
I took [this](https://github.com/googleapis/google-api-python-client) as example

Reply:
Will be implemented in another PR